### DISCUSSION TITLE
mid-control in settings

### DISCUSSION THREAD

**Discussion by DrMacabre** [Mikubill/sd-webui-controlnet#715] [Q&A]
Hello,

what's the purpose of mid-control on hires.fix second pass in the settings ?

**Comment by ljleb** [comment]
The option was added to fix broken generations early in development: https://github.com/Mikubill/sd-webui-controlnet/issues/64#issuecomment-1431592115 I don't know exactly what it does but now you should not need it anymore IIUC to fix the hires pass.
